librz/arch/isa/tricore/tricore_il_fp.inc

```diff
@@ -0,0 +1,591 @@
+// SPDX-FileCopyrightText: 2024 billow <billow.fun@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+static const ut32 HP_MAX_VALUE = 65504;
```

Comment:
Please add a comment what this means, not everyone knows what `HP` is, I guess it's the "half-float", right?